### DISCUSSION TITLE
[Feature] Landing page para gerar convites para o Slack

### DISCUSSION THREAD

**Discussion by mhbarros** [BrasilAPI/BrasilAPI#212] [Ideas]
Atualmente, o convite para o nosso canal no Slack está como um link fixo no README do projeto. Porém, esse link expira a cada 30 dias. 

A ideia é criar uma landing page onde o usuário digitaria o endereço de e-mail e, ao enviar, receberia em sua caixa de entrada um convite / endereço para entrar no workspace.

A ideia inicial é utilizar a lib [rauchg/slackin](https://github.com/rauchg/slackin), com um exemplo demonstrado na comunidade do [BrasilJS](https://braziljs-slack.herokuapp.com/).

**Comment by mhbarros** [comment]
Abri um branch para trabalhar nessa feature, quem quiser colar junto, me dá um toque :)
Aqui o branch: https://github.com/mhbarros/BrasilAPI/tree/landing-slack-invite

**Comment by raphaeldevs** [comment]
Olá, essa feature parece muito útil, como posso contribuir?

**Comment by murilohns** [comment]
@mhbarros, dei uma olhada no slackin e ele precisa de um `Legacy Token` do slack pra funcionar, mas não dá mais pra gerar esse tipo de token lá.

Além disso, dei uma olhada nos `Slack Apps`, e existe uma permissão chamada `admin:invites:write`, que é a necessária para gerar novos invites, porém essa permissão só pode ser utilizada em Slacks com o plano `Enterprise`

**Comment by raphaeldevs** [comment]
@murilohns 😔

**Comment by RodriAndreotti** [comment]
> 
> 
> @murilohns 😔

Acho que o esquema será utilizar realmente a API do Slack para desenvolver algo, eu sempre prefiro partir para o oficial, dependência uma hora ou outra dá dor de cabeça... kkk

**Comment by mhbarros** [comment]
> @mhbarros, dei uma olhada no slackin e ele precisa de um `Legacy Token` do slack pra funcionar, mas não dá mais pra gerar esse tipo de token lá.
> 
> Além disso, dei uma olhada nos `Slack Apps`, e existe uma permissão chamada `admin:invites:write`, que é a necessária para gerar novos invites, porém essa permissão só pode ser utilizada em Slacks com o plano `Enterprise`

Putz, acho que por isso não consegui ir muito a fundo nos testes 😞 
Vou em busca de outra solução, se alguém tiver alguma joga aí !
